### ISSUE TITLE
[Feature Request] Reward and Point

### ISSUE THREAD

**Issue by williamgh2019** [AppCheap/roadmap#101]
It's would be better build into the app with this best free plugin reward and point

https://wordpress.org/plugins/points-and-rewards-for-woocommerce/

Within this plugin as I research and think it's best and free with many features out there such as create Loyalty Programs, Reward Customer Purchases, Referral Points & Redemptions.

**Comment by bakpaorabbit** [comment]
+1 but not for the plugin. looks like the plugin does not support rest api.

[https://woocommerce.com/products/woocommerce-points-and-rewards/](url)

**Comment by ghost** [comment]
the best point and rewards system: 

https://codecanyon.net/item/woocommerce-reward-points/29123909

This really advance system and useful for marketing and sell.

This really should be support..

Thanks

**Comment by sanskritkm** [comment]
> +1 but not for the plugin. looks like the plugin does not support rest api.
> 
> [https://woocommerce.com/products/woocommerce-points-and-rewards/](url)

Good point but as I search in their website said API Setting available in Pro version.
